velox/type/tests/TypeTest.cpp

```diff
@@ -615,6 +615,9 @@ TEST(TypeTest, equality) {
   EXPECT_FALSE(
       *ROW({{"a", INTEGER()}, {"b", REAL()}}) ==
       *ROW({{"a", INTEGER()}, {"d", REAL()}}));
+  EXPECT_FALSE(
```

Comment:
Would you add a test that would catch a bug you had in the original implementation?